benchmarks/dynamo/torchbench.py

```diff
@@ -56,6 +56,7 @@ def setup_torchbench_cwd():
     "hf_T5_base": 4,
     "timm_efficientdet": 1,
     "llama_v2_7b_16h": 1,
+    "yolov3": 8,  # reduced from 16 due to cudagraphs OOM in TorchInductor dashboard
```

Comment:
maybe give a more meaningful name?

Reply:
do you mean renaming the model?

Reply:
wait this is the name of the model?

Reply:
Yes yolov3 is the name of the model: https://github.com/eriklindernoren/PyTorch-YOLOv3